api/filter/http/gzip.proto

```diff
@@ -61,8 +66,9 @@ message Gzip {
   // Array of strings that allows specifying which mime-types yield compression; e.g.
   // application/json, text/html, etc. When this field is not specified, compression will be applied
   // to any "content-type".
-  repeated string content_type = 6
-      [(validate.rules).repeated = {min_items: 0, max_items: 30, unique: true}];
+  // TODO(gsagula) put this back once protoc-gen-validate supports it:
```

Comment:
I'm pretty sure if you remove unique it will work. Can you try that?

Reply:
Removed `unique` on one of them.